pkg/panels/kubernetes/kubelet.go

```diff
@@ -0,0 +1,686 @@
+	panelgroup "github.com/perses/perses/go-sdk/panel-group"
+	statPanel "github.com/perses/perses/go-sdk/panel/stat"
+	timeSeriesPanel "github.com/perses/perses/go-sdk/panel/time-series"
+	"github.com/perses/perses/go-sdk/prometheus/query"
+
+	commonSdk "github.com/perses/perses/go-sdk/common"
+)
+
+func RunningKubeletStat(datasourceName string, labelMatchers ...promql.LabelMatcher) panelgroup.Option {
+	return panelgroup.AddPanel("Running Kubelets",
+		panel.Description("Number of Running Kubelets Instances"),
+		statPanel.Chart(
+			statPanel.Calculation(commonSdk.LastCalculation),
+			statPanel.Format(commonSdk.Format{
+				Unit:          string(commonSdk.DecimalUnit),
+				DecimalPlaces: 0,
+			}),
+		),
+		panel.AddQuery(
+			query.PromQL(
+				promql.SetLabelMatchers(
+					"sum(kubelet_node_name{cluster=~'$cluster',"+GetKubeletMatcher()+"})",
+					labelMatchers,
+				),
+				dashboards.AddQueryDataSource(datasourceName),
+			),
+		),
+	)
+}
+
+func RunningPodStat(datasourceName string, labelMatchers ...promql.LabelMatcher) panelgroup.Option {
+	return panelgroup.AddPanel("Running Pods",
+		panel.Description("Total Number of Running Pods"),
+		statPanel.Chart(
+			statPanel.Calculation(commonSdk.LastCalculation),
+			statPanel.Format(commonSdk.Format{
+				Unit:          string(commonSdk.DecimalUnit),
+				DecimalPlaces: 0,
+			}),
+		),
+		panel.AddQuery(
+			query.PromQL(
+				promql.SetLabelMatchers(
+					"sum(kubelet_running_pods{cluster=~'$cluster',"+GetKubeletMatcher()+", instance=~'$instance'})",
+					labelMatchers,
+				),
+				dashboards.AddQueryDataSource(datasourceName),
+			),
+		),
+	)
+}
+
+func RunningContainersStat(datasourceName string, labelMatchers ...promql.LabelMatcher) panelgroup.Option {
+	return panelgroup.AddPanel("Running Containers",
+		panel.Description("Total Number of Running Containers"),
+		statPanel.Chart(
+			statPanel.Calculation(commonSdk.LastCalculation),
+			statPanel.Format(commonSdk.Format{
+				Unit:          string(commonSdk.DecimalUnit),
+				DecimalPlaces: 0,
+			}),
+		),
+		panel.AddQuery(
+			query.PromQL(
+				promql.SetLabelMatchers(
+					"sum(kubelet_running_containers{cluster=~'$cluster',"+GetKubeletMatcher()+", instance=~'$instance'})",
+					labelMatchers,
+				),
+				dashboards.AddQueryDataSource(datasourceName),
+			),
+		),
+	)
+}
+
+func ActVolumeCountStat(datasourceName string, labelMatchers ...promql.LabelMatcher) panelgroup.Option {
+	return panelgroup.AddPanel("Actual Volume Count",
+		panel.Description("Total Number of Volumes Currently Mounted"),
+		statPanel.Chart(
+			statPanel.Calculation(commonSdk.LastCalculation),
+			statPanel.Format(commonSdk.Format{
+				Unit:          string(commonSdk.DecimalUnit),
+				DecimalPlaces: 0,
+			}),
+		),
+		panel.AddQuery(
+			query.PromQL(
+				promql.SetLabelMatchers(
+					"sum(kubelet_running_containers{cluster=~'$cluster',"+GetKubeletMatcher()+", instance=~'$instance', state='actual_state_of_world'})",
```

Comment:
So this seems like wrong metric, the panel shows up as blank for me, I think it should be
```suggestion
					"sum(volume_manager_total_volumes{cluster=~'$cluster',"+GetKubeletMatcher()+", instance=~'$instance', state='actual_state_of_world'})",
```